docs/polkadot/Polkadot-OverviewPaper/preliminaries.tex

```diff
@@ -18,20 +7,20 @@ \section{Preliminaries}\label{sec:preliminiary}
 %\end{itemize}
 %These points are why we consider Polkadot to be “scalable”. In principle, a problem to be deployed on Polka- dot may be substantially parallelised—scaled out—over a large number of parachains. Since all aspects of each parachain may be conducted in parallel by a different segment of the Polkadot network, the system has some ability to scale. Polkadot provides a rather bare-bones piece of infrastructure leaving much of the complexity to be ad- dressed at the middleware level. This is a conscious decision intended to reduce development risk, enabling the requisite software to be developed within a short time span and with a good level of confidence over its security and robustness.
 
+In this section we describe in more detail the different entities involving in the running of Polkadot, as well as our security model of these entities. This forms the design context for understanding the protocol design described later, including how it works and why it is designed that way.
 
 %\syed{}{perhaps we should use more technical language in this section. I'll try to do so in my review.}
 
 %\paragraph{Chain} (or a blockchain) broadly refers to a state machine which possibly replicated in a decentralized environment. By a block we refer to a recorded list of transactions which have been applied to change the state of a chain.
 
-
 %\paragraph{Relay Chain:} is the Polkadot central state machine which keeps references to the current states of all parachains and to the cross parachain transactions. Thus the relay chain works as a single source of truth across Polkadot network.
 %\syed{It also keeps references of all cross-parachain messages}{-remove- mentioned above as cross parachain transaction, what is a message otherwise? we need to define it first}.
 
 \subsection{Roles}
 Nodes which run Polkadot network assume different roles and functions that we introduce next.
 
 %I'm looking at the definition of these roles in the original paper and in many case they look more comprehensive than what we have here. Perhaps, we could incorporate more of the original white paper here.
-  
+
```

Comment:
Note: these are all whitespace changes, append `?w=1` to the github diff url to ignore, [like this](https://github.com/w3f/research/pull/82/files?w=1)